lib/util/source-code.js

```diff
@@ -117,9 +117,7 @@ function SourceCode(text, ast) {
      */
     this.lines = SourceCode.splitLines(this.text);
 
-    this.tokensAndComments = ast.tokens.concat(ast.comments).sort(function(left, right) {
-        return left.range[0] - right.range[0];
-    });
+    this.tokensAndComments = ast.tokens.concat(ast.comments).sort((left, right) => left.range[0] - right.range[0]);
```

Comment:
This could benefit from being multiline, maybe something like this:

```js
this.tokensAndComments = ast.tokens
    .concat(ast.comments)
    .sort((left, right) => left.range[0] - right.range[0]);
```